Source/WebCore/dom/DocumentMarkerController.h

```diff
@@ -51,6 +51,10 @@ class DocumentMarkerController final : public CanMakeCheckedPtr<DocumentMarkerCo
     WTF_MAKE_FAST_ALLOCATED;
     WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR(DocumentMarkerController);
 public:
+    enum class IterationMode : bool {
+        Forward, Reversed
```

Comment:
Nit - `Forward(s)` and `Backward(s)`, for consistency with other WebKit code.

Maybe `IterationDirection` too, for clarity.